CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- `Dialog, Select`: added `size` prop to documentation, `Dialog` Footer has top border ([@qubis741](https://github.com/qubis741) in [#1993](https://github.com/teamleadercrm/ui/pull/1993))
```

Comment:
It's easier to read if you add them on multiple lines, instead of multiple items on 1 line.